plugins/rbac/src/components/CreateRole/ReviewStepTable.tsx

```diff
@@ -3,17 +3,25 @@ import React from 'react';
 export const ReviewStepTable = ({
   columns,
   rows,
+  tableWrapperWidth,
 }: {
   columns: any[];
   rows: any[];
+  tableWrapperWidth: number;
 }) => {
   return (
-    <div style={{ maxHeight: '230px', overflow: 'scroll', width: '650px' }}>
-      <table style={{ width: '600px' }}>
+    <div
+      style={{
+        maxHeight: '230px',
+        overflow: 'scroll',
```

Comment:
```suggestion
        overflow: 'auto',
```

should this be auto?